boards/riscv32/hifive1/doc/index.rst

```diff
@@ -34,14 +32,8 @@ RISC-V support. Download the tarball for your OS from the `SiFive website
 <https://www.sifive.com/boards>`_ and extract it.
 
 The Zephyr SDK uses a bundled version of OpenOCD by default. You can
-overwrite that behavior by adding the ``OPENOCD`` parameter to the
-``cmake`` command:
-
-.. code-block:: bash
-
-   # run from your build folder
-   cmake -GNinja -DBOARD=hifive1 -DOPENOCD=path/to/riscv-openocd/bin/openocd ..
-
+overwrite that behavior by adding the
+``-DOPENOCD=<path/to/riscv-openocd/bin/openocd>`` parameter when building.
```

Comment:
Doesn't work as-is:

```
$ west build -s zephyr/samples/hello_world/ -d build-test-pr -DTEST_VAR=value
usage: west build [-h] [-b BOARD] [-s SOURCE_DIR] [-d BUILD_DIR] [-t TARGET]
                  [-c] [-f]
                  [cmake_opt [cmake_opt ...]]
west build: error: unexpected arguments: ['-DTEST_VAR=value']
```

Doing it after `--` halts ordinary argument processing and prevents `-D` being parsed as an unrecognized option.

Reply:
Well, this was supposed to be a generic sentence, not about west. I tried to make it vague enough that the user, after reading the corresponding guides, would know how to pass cmake arguments when using cmake directly or west

Reply:
Fixed by adding actual build commands